packages/web-app-files/tests/components/BatchActions.spec.js

```diff
@@ -0,0 +1,232 @@
+import { shallowMount, mount, createLocalVue } from '@vue/test-utils'
+import DesignSystem from 'owncloud-design-system'
+import Vuex from 'vuex'
+import getters from '../../src/store/getters'
+import stubs from '../../../../tests/unit/stubs'
+import BatchActions from '../../src/components/BatchActions.vue'
+
+const localVue = createLocalVue()
+localVue.use(DesignSystem)
+localVue.use(Vuex)
+
+const selectedFiles = ['lorem.txt', 'owncloud.png']
+const componentStubs = { ...stubs, translate: true }
+const elSelector = {
+  copyButton: '#copy-selected-btn',
+  moveButton: '#move-selected-btn',
+  deleteButton: '#delete-selected-btn',
+  restoreButton: 'button.oc-button-primary'
```

Comment:
let's not use utility classes like `oc-button`
if more identifiers are needed, we can add more in the code

Reply:
yeah, that's not the good way but I find it difficult to get a unique identifier for that button. I will look at it more

Reply:
the other buttons have ids, so maybe just add an id also to this button
or a `data-test` property like in https://github.com/owncloud/owncloud-design-system/commit/c6883815810191f0904679d192a39511d1f874e2